### DISCUSSION TITLE
Why is column visibility tie to export funcionality?

### DISCUSSION THREAD

**Discussion by dluccajose** [Laravel-Backpack/community-forum#1185] [Q&A (Help)]
In my mind, these must be two functionalities that could be activated or deactivated individually

**Comment by karandatwani92** [comment]
Hey @dluccajose 

The export & column visibility options that you are using are Datatable's feature. That is why they are tied up with each other.

For a dedicated export operation, you can try:
https://github.com/redsquirrelstudio/laravel-backpack-export-operation
